### ISSUE TITLE
reproduction problem

### ISSUE THREAD

**Issue by wakawaka111** [SCIR-HI/ArtificiallyR2R#1]
Hello, I can't reproduce the experiment result of the paper.I downloaded  the pretained weights from the huggingface hub and finetuned it using ChEBI-20 dataset, finding the result is far away from the result in paper,with the accuracy which is only 0.182.May I ask can you guys upload the weights used in the paper or the pretained weights uploaded is wrong? Thx.

**Comment by haha-super-star** [comment]
I have the same problem, too. During fine-tuning, all the hyperparameters I set are consistent with those mentioned in the paper with the learning rate of 1e-3, 50,000 fine-tuning steps, weight decay of 0.1, batch size of 32, and 1000 warm-up steps.

**Comment by XingyuLu206** [comment]
My reproduction is successful with small size, but failed with base size, I use the provided code and ChEBI data, without changing any hyper parameters.

**Comment by Imsovegetable** [comment]
The pre-trained weights we uploaded is correct. It seems you guys encounter the model collapse during training, which can catastrophically affect model performance. Please check the loss curve and see if there is a sudden increase in loss during training. In our experiments, this may happen in the first 20k steps. 

In addition, we find that the base size is more prone to this situation than small size models, especially the base size model that has been pre-trained twice (we guess that 1e-3 may be a bit too large for the t5-base that has been pre-trained twice). Since previous research work selected 1e-3, we chose to remain consistent. 

If this situation continues to occur, you can try to use a smaller learning rate, such as 5e-4 or 3e-4, which will greatly increase training stability and will not greatly affect model performance.

**Comment by XingyuLu206** [comment]
Thank you very much for your advice, I will try to lower the lr as soon as possible.
